### ISSUE TITLE
[BUG] - special (repeated) characters are not escaped and are interpreted

### ISSUE THREAD

**Issue by dgrebb** [strapi/strapi-plugin-seo#43]
I haven't tested this in multiple fields, but when adding, for example, "C++" to the keywords field, JavaScript attempts to interpret the string as code.

This may be a security concern.

`Uncaught (in promise) SyntaxError: nothing to repeat`

somewhere in here:

```javascript
zr = (Nt, Ln) => {
  Nt.map(
    Nn => {
      if (!Z().isEmpty(Nn)) {
        const Tr = Nn.trim(),
        mr = new RegExp(Tr, 'g'),
        Nr = (Ln.join(' ').match(mr) || []).length;
        Xt[Tr] === void 0 ? Xt[Tr] = {
          count: Nr
        }
         : Xt[Tr].count += Nr
      }
    }
  )
},
```

Upon removing "C++" from the keywords field, the error is gone.

Happy to help debug! Not a show stopper for me, only an observation.

**Comment by Mcastres** [comment]
Hum interesting 🤔 
Thanks for the heads up! I will take a closer look at this 👍

**Comment by Mcastres** [comment]
Fixed in 1.9.2 🔥
